BridgeAppSDK/SBAUserWrapper.swift

```diff
@@ -65,11 +65,16 @@ public protocol SBAUserWrapper: class, SBBAuthManagerDelegateProtocol {
     var password: String? { get set }
     
     /**
-     * Gender is stored in the keychain
+     * Gender describes the characteristics that a society or culture delineates as masculine or feminine. Stored in the keychain.
```

Comment:
Good point, though I would prefer to let the actual language of the questions wait for Woody and Christine's input and for now just use the terms defined in ResearchKit.